manifest/deployment.yaml

```diff
@@ -13,8 +19,8 @@ spec:
         run: tagtodigest-provider
     spec:
       containers:
-      - image: sozercan/tagtodigest-provider:v0.0.1
-        imagePullPolicy: Always
+      - image: tagtodigest-provider:latest
```

Comment:
ideally, we want this to work out of box. we can cut a release and push afterwards